src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -30,10 +30,13 @@ public static void ConcurrentInitializeTest()
                 {
                     countdownEvent.Signal();
                     countdownEvent.Wait(ThreadTestHelpers.UnexpectedTimeoutMilliseconds);
-                    Assert.True(ThreadPool.SetMinThreads(processorCount, processorCount));
                 };
 
+            // ThreadPool.SetMinThreads returns false on UapAot
+            ThreadPool.SetMinThreads(processorCount, processorCount);
```

Comment:
The purpose of this test is to call SetMinThreads concurrently from different threads, so SetMinThreads needs to be called from the background thread. If SetMinThreads returns false on uapaot, what is wrong with this test that requires a change?